code/index.html

```diff
@@ -3,28 +3,187 @@
+    <title>Winecellar</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Stardos+Stencil&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/37c653cfef.js" crossorigin="anonymous"></script>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+
+    <!-- Header -->
+    <header class="header">
+
+      <picture>
+        <source srcset="./images/michael-oeser-PCxvRoNoi6w-unsplash.jpg" media="(max-width: 767px)" alt="wineglass" />
+        <source srcset="./images/mae-mu-rgEHTLwaI_8-unsplash.jpg" media="(max-width: 1199px)" alt="winebottle" />
+        <img class="image" src="./images/joost-crop-E5Ne2TfbTFw-unsplash.jpg" alt="Winebottle" />
+
+        <nav class="navigation-container">
+          <a id="about-us-button" class="navigation-links">about</a>
+          <a id="contact-us-button" class="navigation-links">contact</a>
+        </nav>
+        
+        <div class="title-container">
+          <h1 class="h1">winecellar</h1>
+          <h2 class="h2">book your private wine tasting experience</h2>
+          <div class="book-now-button" id="book-now">BOOK NOW</div>
+        </div>
+      </picture>
+
+    </header>
+
+    <!-- Sign up Form and Accordion -->
+
+    <div class="big-content-container" id="big-content">
 
     <!-- Signup form -->
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="form-container">
+
+        <h3 class="header-container">book wine tasting</h3>
+
+        <form class="form" action="https://httpbin.org/anything" method="post">
+          <div class="form-question">
+            <label for="name">name:</label>
+            <input type="text" id="name" class="input-text" name="name" placeholder="Enter your full name" required>
+          </div>
+
+          <div class="form-question">
+            <label for="email">e-mail:</label>
+            <input type="email" id="email" class="input-text" name="email" placeholder="Your e-mail address goes here" required>
+          </div>
+
+          <div class="form-question">
+            <label for="number">phone number:</label>
+            <input type="tel" id="number" class="input-text" name="phone-number" placeholder="Your phone number goes here" required>
+          </div>
+
+          <div class="date-and-number">
+            <div class="form-question">
+              <label for="date">when do you want to book?</label>
+              <input type="date" id="date" name="date" min="2021-01-31" max="2022-01-31" required>
+            </div>
+
+            <div class="form-question">
+              <label for="people">how many?</label>
+              <select class="form-number-of-people" id="people" name="number-of-people" required>
+                <option selected>0</option>
```

Comment:
You can add the `disabled` attribute to prevent the user from selecting this one. Instead of `0` you could write a placeholder